### ISSUE TITLE
New headless chrome crashes when passing "--single-process"

### ISSUE THREAD

**Issue by danielgindi** [hardkoded/puppeteer-sharp#2512]
### Description

When passing `--single-process` - the headless chrome crashes. It worked before the new headless version.

### Complete minimal example reproducing the issue

Complete means the code snippet can be copied into a unit test method in a fresh C# project and run.
Minimal means it is stripped from code not related to reproducing the issue.

E.g.

```csharp

                                  var browser= await PuppeteerSharp.Puppeteer.LaunchAsync(new PuppeteerSharp.LaunchOptions
                                  {
                                      Args = new string[] {
                                          "--single-process", // Less processes -- Currently crashes chrome
                                      },
                                      Headless = true,
                                  });
```
.
### Expected behavior:

`chrome.exe` should prevail.

### Actual behavior:

`chrome.exe` crashes.

### Versions

15.0.0, 15.1.0
.NET 7.0

### Additional Information

When calling the new `chrome.exe` from cmd line with --single-process - it also crashes.

**Comment by kblok** [comment]
I think that `--single-process` was more a `chromium` flag than a `chrome` one. You could try using `HeadlessMode = HeadlessMode.Shell` to use the legacy browser or use the `SupportedBrowser.Chromium` instead of `Chrome`.

**Comment by danielgindi** [comment]
Oh so we have moved from Chromium to Chrome? I did not even notice. Will the new headless mode work with chromium too?

**Comment by kblok** [comment]
> Oh so we have moved from Chromium to Chrome? I did not even notice. Will the new headless mode work with chromium too?

Yes! Since [v11](https://github.com/hardkoded/puppeteer-sharp/releases/tag/v11.0.0). The new headless mode is prior to Chrome for testing in puppeteer. So I bet it works in chromium. That flag god renamed a few times, so that's why I'm not 100% sure.

**Comment by danielgindi** [comment]
Since moving to Chrome I've experienced some instabilities. Occasionally a 100% cpu usage, and infinite logging of GPU errors. I had to pin the build number to stabilize it. Looks like Chromium is far more stable.

**Comment by kblok** [comment]
Closed due to inactivity. Feel free to reopen it if needed.
